packages/klingon-ui/src/app/_shared/terminal/terminal.service.ts

```diff
@@ -100,6 +100,10 @@ export class TerminalService {
     return this.cli.runNgCommand(data);
   }
 
+  command(data) {
+    this.term.send(`${data}\n`);
```

Comment:
Almost done. Just working on communicating server side validation errors with ui.

Reply:
> I am afraid this may introduce an XSS injection breach. Users can send any malicious commands. Can we add some guards to prevent this? Maybe secure the server side?

Vallidation added. @manekinekko